ref/net46/Microsoft.Build.Tasks.Core/Microsoft.Build.Tasks.Core.cs

```diff
@@ -2503,6 +2503,71 @@ public abstract partial class XamlDataDrivenToolTask : Microsoft.Build.Utilities
         protected override bool ValidateParameters() { throw null; }
     }
 }
+namespace Microsoft.VisualStudio.Setup.Configuration
```

Comment:
We don't ship the assembly, they're just embedded interop types for the COM API. There's a feature request on Roslyn to `/linkinternal` rather than `/link`, but we have no choice right now they have to be public.